doc/content/reference/api/_index.md

```diff
@@ -16,9 +16,176 @@ This is the reference for the gRPC and HTTP APIs that {{% tts %}} exposes.
 Additional fields may be specified in HTTP requests by appending them as query string parameters. For example, to request the `name`, `description`, and `locations` of devices in an `EndDeviceRegistry.Get` request, add these fields to the `field_mask` field. To get this data for device `dev1` in application `app1`:
 
 ```bash
-curl -i -H "Authorization: Bearer NNSXS.XXXXXXXXX" https://thethings.example.com/api/v3/applications/app1/devices/dev1?field_mask=name,description,locations
+curl --location --header "Authorization: Bearer NNSXS.XXXXXXXXX" https://thethings.example.com/api/v3/applications/app1/devices/dev1?field_mask=name,description,locations
 ```
 
+The same request in tenant `tenant1` on a multi-tenant instance:
+
+```bash
+curl --location --header "Authorization: Bearer NNSXS.XXXXXXXXX" https://tenant1.thethings.example.com/api/v3/applications/app1/devices/dev1?field_mask=name,description,locations
+```
+
+Fields may also be specified as a JSON object in a POST request.
+
+To get a list of events for device `dev1` in application `app1` :
+
+```bash
+curl --location \
+     --header 'Authorization: Bearer NNSXS.XXXXXXXXX' \
+     --header 'Content-Type: text/plain' \
```

Comment:
The content type of the request is definitely not `text/plain`. We're sending JSON.

Reply:
And we're expecting JSON back, so we should probably also have `Accept` for JSON.